RetsTalk/RetsTalk/RetrospectCalendar/Controller/RetrospectCalendarViewController.swift

```diff
@@ -152,9 +124,48 @@ extension RetrospectCalendarViewController: @preconcurrency UICalendarSelectionS
         guard let dateComponents = dateComponents else { return }
```

Comment:
다음과 같이 줄일 수 있는 것으로 알고있습니다!
```suggestion
        guard let dateComponents else { return }
```